app/src/main/res/layout/list_medicalhistory_hypertension_treatment.xml

```diff
@@ -8,15 +8,14 @@
 
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="wrap_content"
+    android:padding="@dimen/spacing_16">
 
     <TextView
       android:id="@+id/labelTextView"
       android:layout_width="0dp"
```

Comment:
Not important to change here but we are using hardcoded dimen value here

Reply:
That change is not from this PR, we will removing this file anyway.